lib/dpl/provider/surge.rb

```diff
@@ -0,0 +1,34 @@
+module DPL
+  class Provider
+    class Surge < Provider
+      npm_g 'surge'
+      context.shell "git rev-parse && cd \"$(git rev-parse --show-toplevel)\"" # Go to repo root 
```

Comment:
Furthermore, I am not sure if this is the right place to change directories. This part is executed when the file is loaded, which doesn't seem a good idea.